rpa-builder/6.4/modules/ROOT/pages/toolbox-variable-handling-json-query.adoc

```diff
@@ -1,48 +1,49 @@
 
-
 = Json Query
 
-The _Json Query_ Action Step enables you to parse Json objects and to
-read individual values / elements from the object. The result can then
-be used in the remainder of the process run as a source variable.
+Use the Json Query Action Step to parse Json objects and read individual values or elements from the object. The result remains available for the rest of the process run as a source variable.

```

Comment:
Global: Json > JSON

Reply:
The result is available for the rest of the process to use as a source variable.

Even better - The rest of the process can use the result as a source variable.